tests/test_polytracker.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import os
+from polyprocess import PolyProcess
+
+cwd: str = os.getcwd()
+test_dir: str = cwd + "/tests/"
+bin_dir: str = test_dir + "/bin/"
+test_results_dir: str = bin_dir + "/test_results/"
+bitcode_dir: str = test_dir + "/bitcode/"
+
+"""
+Pytest fixture to init testing env (building tests) 
+
+This runs before any test is executed
+"""
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_targets():
+    # Check if bin dir exists
+    if os.path.exists(bin_dir):
+        os.system("rm -r " + bin_dir)
```

Comment:
This will fail if the path to `bin_dir` contains a space. You should probably change all of the `os.system` calls in here to `subprocess.call([arg1, arg2, arg3, ...])`.

Reply:
Ahh yeah, I forgot I have os.system still in the test files. I will replace it all with subprocess.call like I did in polybuild

Reply:
Done!